snippets/global.json

```diff
@@ -1,9 +1,29 @@
 {
+  "copyright": {
+    "prefix": "c)",
+    "body": ["Copyright (c) ${CURRENT_YEAR} ${0:Author}. All Rights Reserved."],
+    "description": "Snippet to put copyright"
+  },
   "diso": {
     "prefix": "diso",
     "body": ["${CURRENT_YEAR}-${CURRENT_MONTH}-${CURRENT_DATE}T${CURRENT_HOUR}:${CURRENT_MINUTE}:${CURRENT_SECOND}"],
     "description": "ISO date time stamp"
   },
+  "date": {
+    "prefix": "date",
+    "body": ["${CURRENT_YEAR}-${CURRENT_MONTH}-${CURRENT_DATE}"],
+    "description": "Put the date in (Y-m-D) format"
+  },
+  "time": {
+    "prefix": "time",
+    "body": ["${CURRENT_HOUR}-${CURRENT_MINUTE}"],
+    "description": "I give you back the time (H-M)"
```

Comment:
Would it be possible to get the format for the time to use a colon instead of the hyphen?

Reply:
Yes, I think it's better to use the colon.